db/versions/0061.yml

```diff
@@ -0,0 +1,395 @@
+version: 61
+description: Deprecate functions that use provisioner_id/worker_type and add equivalents using task_queue_id instead, for tasks table
+methods:
+  create_task:
```

Comment:
From what I can see, this is the same function from version 0060, but with `deprecated: true`.  As a shortcut, you can omit everything except `deprecated` for such changes.